Projects/Server/Mobiles/Mobile.cs

```diff
@@ -285,6 +285,12 @@ public enum StatLockType : byte
         Locked
     }
 
+    public enum PossessType : byte
```

Comment:
Are there other types of possession to consider? I am wondering if we should just use a bool.

Reply:
Agreed, I started off with `None`, `Possessed` and `Possessing` initially but realized keeping the `Possessing` state is unnecessary.